plugins/loading/rollup.config.js

```diff
@@ -31,7 +30,7 @@ const config = {
 		}, // Universal Modules
 		{ file: pkg.main, format: 'cjs', exports: 'named', sourcemap: true }, // CommonJS Modules
 		{ file: pkg.module, format: 'es', exports: 'named', sourcemap: true }, // ES Modules
-		{ file: pkg.types, format: 'es' },
+		// { file: pkg.types, format: 'es' },
```

Comment:
remove?

Reply:
@ShMcK Type files cannot be output here.

Reply:
@ShMcK I deleted it.